src/datepicker/index.es

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+
+import keymanager from 'nebenan-helpers/lib/keymanager';
+import eventproxy from 'nebenan-helpers/lib/eventproxy';
+
+import Picker from './calendar';
```

Comment:
Workaround for local development so that I don't have to republish nebenan-react-datepicker all the time.